config/development.js

```diff
@@ -13,8 +13,8 @@ module.exports = development = {
 	DEV: true,
 	OTP_FIXED : 1234,
     EMAIL_HOST: "sudhansu@ith.tech",
-    MAILJET_APIKEY_PUBLIC: 'c47de902fdfd2f36c8ec79e2613f5d73', // ScriptToAddContacts : UserAccountName
-    MAILJET_APIKEY_PRIVATE: '2a6501a2791deb1ea480dc0ec69a55ea', // ScriptToAddContacts : UserAccountName
```

Comment:
i'll get this changed as soon as possible this mailjet key was inserted to test if this key is working or not